index.html

```diff
@@ -3308,10 +3312,16 @@
         Kanai, Tobie Langel, Tomoyuki Shimizu, Travis Leithead, and Wayne Carr
         for help with editing, reviews and feedback to this draft.
       </p>
+      <p>
+        <em>AirPlay</em>, <em>HDMI</em>, <em>Chromecast</em>, <em>DLNA</em> and
```

Comment:
I checked internally. Specs we publish usually do not contain non-W3C trademarks for various reasons. In our case, mentioning these technologies as examples in the introduction is important to clarify the context for readers, and the introduction is clear that other technologies are available.

It is a good idea to call these trademarks out explicitly in the acknowledgement section. The proposed text is fine. I recommend adding "They are only cited as background information" to the end of it to make it crystal clear that one does not need them to implement the specification.